bob/src/hw_metrics_collector.rs

```diff
@@ -16,7 +18,9 @@ const DESCRS_DIR: &str = "/proc/self/fd/";
 const CPU_STAT_FILE: &str = "/proc/stat";
 const DISK_STAT_FILE: &str = "/proc/diskstats";
 
+#[derive(Debug, Clone)]
 pub(crate) struct DiskSpaceMetrics {
+    pub(crate) disk_name: DiskName,
```

Comment:
It's not a good idea to put the disk_name filed in this struct, since the mapping is ambiguous (multiple bob disks can be placed on single physical disk). Also, this field will not be used anywhere after the corrections will be made accrding to my previous comments. Thus I would suggest to remove this field